### PR TITLE
NOTICK - Add Trademarks file

### PR DESCRIPTION
**Pull request by fenryka**

### CODE REVIEW

File: buildSrc/src/main/groovy/DeployableContainerBuilder.groovy [hunk 1]
```diff
@@ -161,8 +161,8 @@ abstract class DeployableContainerBuilder extends DefaultTask {
             if (imageName.endsWith("-local")) {
                 logger.info("Resolving base image ${baseImageName.get()}:${baseImageTag.get()} from local Docker daemon")
                 builder = Jib.from(DockerDaemonImage.named(imageName))
-            } else if (imageName.contains("corda-os-cli")) {
-                logger.info("Resolving base image ${baseImageName.get()}: ${baseImageTag.get()} from internal remote repo")
+            } else if (imageName.contains("software.r3.com")) {
+                logger.info("Resolving base image ${baseImageName.get()}:${baseImageTag.get()} from internal remote repo")
```
Comment (driessamyn) [inline buildSrc/src/main/groovy/DeployableContainerBuilder.groovy]: Is this change intentional?

Comment (fenryka) [reply buildSrc/src/main/groovy/DeployableContainerBuilder.groovy]: No.... that's odd, looking at my branch locally that change isn't there

git diff release/os/5.0
diff --git a/TRADEMARK b/TRADEMARK
new file mode 100644
index 000000000..aa2799e5d
--- /dev/null
+++ b/TRADEMARK
@@ -0,0 +1,4 @@
+Corda and the Corda logo are trademarks of R3 HoldCo LLC and its affiliates.
+All rights reserved.
+
+For R3 HoldCo LLC's trademark and logo usage information, please consult our Trademark Usage Policy available at https://www.r3.com/trademark-policy

will rebase the PR
